.fixtures.yml

```diff
@@ -1,13 +1,11 @@
 fixtures:
   repositories:
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
+    apt: https://github.com/puppetlabs/puppetlabs-apt
   forge_modules:
     inifile:
```

Comment:
I really think the modules shouldn't be pinned here. IMO best practice is to test against main branches in unit tests and against forge releases in acceptance tests.

Reply:
Agreed, the versions are almost always going to be out of date, and puppetlabs/translate shouldn't even be there anymore. I would remove this commit before merging this, but I wanted to see if would run with puppetlabs/apt main. Looks like they did and I have some tests to fix up.

Reply:
Made separate PR #683 for this.